crates/vm/levm/src/opcode_handlers/environment.rs

```diff
@@ -424,10 +424,17 @@ impl VM {
 
         self.increase_consumed_gas(current_call_frame, gas_cost::extcodehash(address_was_cold)?)?;
 
-        current_call_frame.stack.push(U256::from_big_endian(
-            keccak(account_info.bytecode).as_fixed_bytes(),
-        ))?;
+        // An account is considered empty when it has no code and zero nonce and zero balance. [EIP-161]
+        if account_info.bytecode.is_empty()
+            && account_info.nonce == 0
+            && account_info.balance == U256::zero()
```

Comment:
```suggestion
        if account_info.is_empty()
```
We already have a method in `AccountInfo` for that :)

Reply:
That's nice! Done [749116f](https://github.com/lambdaclass/ethrex/pull/1503/commits/749116fd23e196720c49ea1184ac8b115afc8de7)